llvm/tools/llvm-link/llvm-link.cpp

```diff
@@ -479,6 +480,9 @@ int main(int argc, char **argv) {
 
   cl::HideUnrelatedOptions({&LinkCategory, &getColorCategory()});
   cl::ParseCommandLineOptions(argc, argv, "llvm linker\n");
+  // Load bitcode into the new debug info format by default.
```

Comment:
Ultra-nit: Maybe add a blank above this to separate it from the "default" parsing step.